system/tests/kohana/ORMTest.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
```

Comment:
Missing docblock. ORM is not part of core, so this file should be placed in /modules/orm/tests/

Reply:
I've moved the test to the location requested, and added a docblock based on others I could see around the project